frontend/src/components/ViewActivities/activityday.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Accordion from 'react-bootstrap/Accordion';
+import Card from 'react-bootstrap/Card';
+import Activity from './activity.js';
+import * as activityFns from './activityfns.js';
+import * as utils from '../Utils/utils.js'
+
+class ActivityDay extends React.Component {
+  /** @inheritdoc */
+  render() {
+    const sortedActivities = 
+        Array.from(this.props.activities).sort(activityFns.compareActivities);
+    let date = new Date(this.props.date);
+    let id = date.getTime();
```

Comment:
I'm pretty sure you can make these const.
```suggestion
    const date = new Date(this.props.date);
    const id = date.getTime();
```